python/cugraph/dask/pagerank/pagerank.py

```diff
@@ -278,3 +282,32 @@ def get_chunksize(input_path):
         size = [os.path.getsize(_file) for _file in input_files]
         chunksize = max(size)
     return chunksize
+
+
+def _read_csv(input_files, delimiter='\t', names=['src', 'dst'],
+              dtype=['int32', 'int32']):
+    df = []
+    for f in input_files:
+        df.append(cudf.read_csv(f, delimiter=delimiter, names=names,
+                                dtype=dtype))
+    df_concatenated = cudf.concat(df)
+    return df_concatenated
+
+
+def read_split_csv(input_files):
+    client = default_client()
+    n_files = len(input_files)
+    n_gpus = get_n_gpus()
+    n_files_per_gpu = int(n_files/n_gpus)
+    worker_map = []
+    for i, w in enumerate(client.has_what().keys()):
+        files_per_gpu = input_files[i*n_files_per_gpu: (i+1)*n_files_per_gpu]
+        worker_map.append((files_per_gpu, w))
+    t0 = time.time()
+    new_ddf = [client.submit(_read_csv, part, workers=[worker])
+               for part, worker in worker_map]
+
+    wait(new_ddf)
+    t1 = time.time()
+    print("Reading Csv time: ", t1-t0)
```

Comment:
the timer should not be inside the API function

Reply:
moved it in the test